compiler-core/src/language_server/compiler.rs

```diff
@@ -114,6 +114,22 @@ where
             _ = self.sources.insert(module.name.clone(), source);
         }
 
+        // Since cached modules are not recompiled we need to manually add them
+        for (name, module) in self.project_compiler.get_importable_modules() {
+            if self.sources.contains_key(name) || name == "gleam" {
+                continue;
+            }
+            // Get the build path for the module
+            let build_path = self
+                .project_compiler
+                .get_build_dir_for_module(&module.package, &module.name);
+            // Create the source information
+            let path = build_path.as_os_str().to_string_lossy().to_string();
```

Comment:
This is a utf8 path so should be able to do just `.to_string()`, no?

Reply:
yup looks like this was a copy paste on my end. will fix